nxc/modules/schtask_as.py

```diff
@@ -18,14 +18,26 @@ def options(self, context, module_options):
         """
         CMD            Command to execute
         USER           User to execute command as
+        TASK           Set a name for the scheduled task name
+        FILE           Set a name for the command output file
+        LOCATION       Set a location for the command output file
```

Comment:
Please also add the default, so the user knows these must not always be set

Reply:
Updated to add "optional" to the flags